mews-operations/availabilityBlockProcessor.md

```diff
@@ -0,0 +1,176 @@
+| `releaseStrategy`   | [`releaseStrategy`](#Release-Strategy) object                        | required              | Release strategy of the availability block.                                                                                      |
+| `originalRateCode`    | `string`                                                            | required              | Original rated code of the availability block.                                                                                   |
+| `spaceCategories`    | [`Space category allocation`](#space-category-allocation) collection | required              | Allocated categories of the availability block.                                                                                  |
+| `booker`             | [`Customer`](./reservations.md#customer) object                      | required              | The main booker. This does not mean that the person has arrived at the property.                                                 |
+| `company`            | [`Company`](./reservations.md#company) object                        | optional              | The company associated with the availability block.                                                                              |
+| `notes`              | `string`                                                             | optional              | It's a notes.                                                                                                                    |
+
+
+#### Dates
+| Property | Type | Contract | Description                                                                                                                                                                          |
+| :-- | :-- | :-- |:-------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| `start` | `string` | required | Start date in format `yyyy-mm-dd`, e.g. `2024-12-05`; this is the first day of the block and so can be booked for stay.                                                              |
+| `end` | `string` | required | End date in format `yyyy-mm-dd`, e.g. `2024-12-08`; this day is _not_ included in the block and so cannot be booked for stay (it is analogous to reservation departure).             |
+
+> **Dates**: Interval for 1 night (e.g. is represented 2024-12-24/25 is represented as `"start": "2024-12-24", "end": "2024-12-24"`).
+
+
+#### Release strategy
+| Property | Type | Contract | Description                                                                                  |
+| :-- | :-- | :-- |:---------------------------------------------------------------------------------------------|
+| `type` | `string` | required | release strategy for the availability block, in the first version only support "fixed" |
```

Comment:
enums should have also type in documentation so when you extend it there is link for it